tests/test_unit/test_features_generators.py

```diff
@@ -0,0 +1,145 @@
+import pytest
+from chemprop.features import features_generators
+from rdkit import Chem
+from typing import Sequence
+
+TEST_SMILES_STRING = "C"
+TEST_MOLECULE_OBJECT = Chem.MolFromSmiles(TEST_SMILES_STRING)
+
+TEST_MULTIMOLECULE_LIST = [
+    [TEST_SMILES_STRING, TEST_SMILES_STRING],
+    [TEST_SMILES_STRING, TEST_SMILES_STRING],
+]
+
+MORGAN_RADIUS = 2
+MORGAN_NUM_BITS = 2048
+
+NUM_RDKIT_FEATURES = 200
+
+
+# Ensure the output dimensionality of each generator is correct
+class TestFeatureGenerators:
```

Comment:
why are all the tests inside a stateless class?

Reply:
No reason. Just now realized that tests don't need to be in a class like they do if we were using unittests. Will remove.

Reply:
Ignore my last comment. They're to help organize the different types of tests in the file (namely tests that test the feature generators themselves and tests that test the registration of generators). There's a second class in the file TestGeneratorRegistration.